airbyte-integrations/connectors/source-zendesk-support/source_zendesk_support/streams.py

```diff
@@ -504,32 +507,29 @@ def stream_slices(
         self, sync_mode: SyncMode, cursor_field: Optional[List[str]] = None, stream_state: Optional[Mapping[str, Any]] = None
     ) -> Iterable[Optional[Mapping[str, Any]]]:
         parent_stream_state = None
-        if stream_state:
-            cursor_value = stream_state.get(self.cursor_field)
-            parent_stream_state = {self.parent.cursor_field: pendulum.parse(cursor_value).int_timestamp}
-        else:
-            cursor_value = self._start_date
-
+        cursor_value = (stream_state or {}).get(self.cursor_field, pendulum.parse(self._start_date).int_timestamp)
+        parent_stream_state = {self.parent.cursor_field: cursor_value}
         parent_records = self.parent.read_records(
             sync_mode=SyncMode.incremental, cursor_field=cursor_field, stream_slice=None, stream_state=parent_stream_state
         )
 
         for record in parent_records:
-            yield {"ticket_id": record["id"], self.cursor_field: cursor_value}
```

Comment:
i'm surprised this worked. We're not even populating the parent record's respective cursor value when we generate slices...

How you now have it probably makes more sense

Reply:
It was passed down to `TicketMetrics.parse_response` to filter records.

Previously:
```python
            cursor_date = (stream_slice or {}).get(self.cursor_field)
            updated = data[self.cursor_field]
            if not cursor_date or updated >= cursor_date:
                yield data
```